server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxSetMethodContract.scala

```diff
@@ -1772,6 +1773,48 @@ trait MailboxSetMethodContract {
          |}""".stripMargin)
   }
 
+  @Disabled("TODO implementmailbox/set creation subscription support and enable upon rebase")
```

Comment:
Is that for an other ticket/PR?

Reply:
See https://github.com/linagora/james-project/pull/3661